src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { People } from './components/PeoplePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="home" element={<Navigate to="/" />} />
+        <Route index element={<h1 className="title">Home Page</h1>} />
+
```

Comment:
Would be good to create separate component for this

Reply:
Good point, fix it